Can not compile discord rs with chrono 0.4

After recent update of `chrono` crate I can't compile the discord crate anymore:

```
   Compiling discord v0.8.0 (https://github.com/SpaceManiac/discord-rs#e401c10b)
error[E0277]: the trait bound `chrono::DateTime<chrono::FixedOffset>: serde::Serialize` is not satisfied
   --> /home/user/.cargo/git/checkouts/discord-rs-e20ed8881d23c413/e401c10/src/model.rs:389:24
    |
389 | #[derive(Debug, Clone, Serialize, Deserialize)]
    |                        ^^^^^^^^^ the trait `serde::Serialize` is not implemented for `chrono::DateTime<chrono::FixedOffset>`
    |
    = note: required because of the requirements on the impl of `serde::Serialize` for `std::option::Option<chrono::DateTime<chrono::FixedOffset>>`
    = note: required by `serde::ser::SerializeStruct::serialize_field`

error[E0277]: the trait bound `chrono::DateTime<chrono::FixedOffset>: serde::Deserialize<'_>` is not satisfied
   --> /home/user/.cargo/git/checkouts/discord-rs-e20ed8881d23c413/e401c10/src/model.rs:389:35
    |
389 | #[derive(Debug, Clone, Serialize, Deserialize)]
    |                                   ^^^^^^^^^^^ the trait `serde::Deserialize<'_>` is not implemented for `chrono::DateTime<chrono::FixedOffset>`
    |
    = note: required because of the requirements on the impl of `serde::Deserialize<'_>` for `std::option::Option<chrono::DateTime<chrono::FixedOffset>>`
    = note: required by `serde::de::SeqAccess::next_element`

error[E0277]: the trait bound `chrono::DateTime<chrono::FixedOffset>: serde::Deserialize<'_>` is not satisfied
   --> /home/user/.cargo/git/checkouts/discord-rs-e20ed8881d23c413/e401c10/src/model.rs:389:35
    |
389 | #[derive(Debug, Clone, Serialize, Deserialize)]
    |                                   ^^^^^^^^^^^ the trait `serde::Deserialize<'_>` is not implemented for `chrono::DateTime<chrono::FixedOffset>`
    |
    = note: required because of the requirements on the impl of `serde::Deserialize<'_>` for `std::option::Option<chrono::DateTime<chrono::FixedOffset>>`
    = note: required by `serde::de::MapAccess::next_value`

error[E0277]: the trait bound `chrono::DateTime<chrono::FixedOffset>: serde::Serialize` is not satisfied
   --> /home/user/.cargo/git/checkouts/discord-rs-e20ed8881d23c413/e401c10/src/model.rs:647:24
    |
647 | #[derive(Debug, Clone, Serialize, Deserialize)]
    |                        ^^^^^^^^^ the trait `serde::Serialize` is not implemented for `chrono::DateTime<chrono::FixedOffset>`
    |
    = note: required by `serde::ser::SerializeStruct::serialize_field`

error[E0277]: the trait bound `chrono::DateTime<chrono::FixedOffset>: serde::Deserialize<'_>` is not satisfied
   --> /home/user/.cargo/git/checkouts/discord-rs-e20ed8881d23c413/e401c10/src/model.rs:647:35
    |
647 | #[derive(Debug, Clone, Serialize, Deserialize)]
    |                                   ^^^^^^^^^^^ the trait `serde::Deserialize<'_>` is not implemented for `chrono::DateTime<chrono::FixedOffset>`
    |
    = note: required by `serde::de::SeqAccess::next_element`

error[E0277]: the trait bound `chrono::DateTime<chrono::FixedOffset>: serde::Deserialize<'_>` is not satisfied
   --> /home/user/.cargo/git/checkouts/discord-rs-e20ed8881d23c413/e401c10/src/model.rs:647:35
    |
647 | #[derive(Debug, Clone, Serialize, Deserialize)]
    |                                   ^^^^^^^^^^^ the trait `serde::Deserialize<'_>` is not implemented for `chrono::DateTime<chrono::FixedOffset>`
    |
    = note: required by `serde::de::MapAccess::next_value`

error: aborting due to 6 previous errors

error: Could not compile `discord`.
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -30,7 +30,7 @@ flate2 = "0.2.14"
 opus = { version = "0.2", optional = true }
 
 [dependencies.chrono]
-version = "0.3"
+version = "0.4"
 features = ["serde"]
 
 [dependencies.sodiumoxide]

--- a/src/builders.rs
+++ b/src/builders.rs
@@ -5,8 +5,8 @@
 
 use serde_json::Value;
 
-use chrono::offset::fixed::FixedOffset;
-use chrono::datetime::DateTime;
+use chrono::offset::FixedOffset;
+use chrono::DateTime;
 
 use model::*;
 use Object;

--- a/src/model.rs
+++ b/src/model.rs
@@ -9,9 +9,9 @@ use serde_json::Value;
 
 use super::{Error, Result, Object};
 
-use chrono::datetime::DateTime;
-use chrono::offset::fixed::FixedOffset;
-use chrono::offset::utc::UTC;
+use chrono::DateTime;
+use chrono::FixedOffset;
+use chrono::offset::Utc;
 use chrono::offset::TimeZone;
 
 pub use self::permissions::Permissions;
@@ -1502,7 +1502,7 @@ pub enum Event {
 	TypingStart {
 		channel_id: ChannelId,
 		user_id: UserId,
-		timestamp: DateTime<UTC>,
+		timestamp: DateTime<Utc>,
 	},
 	/// A member's presence state (or username or avatar) has changed
 	PresenceUpdate {
@@ -1688,7 +1688,7 @@ impl Event {
 			warn_json!(value, Event::TypingStart {
 				channel_id: try!(remove(&mut value, "channel_id").and_then(ChannelId::decode)),
 				user_id: try!(remove(&mut value, "user_id").and_then(UserId::decode)),
-				timestamp: UTC.timestamp(req!(try!(remove(&mut value, "timestamp")).as_i64()), 0),
+				timestamp: Utc.timestamp(req!(try!(remove(&mut value, "timestamp")).as_i64()), 0),
 			})
 		} else if kind == "PRESENCE_UPDATE" {
 			let server_id = try!(opt(&mut value, "guild_id", ServerId::decode));

